Cargo.toml

```diff
@@ -54,10 +51,18 @@ opus = "0.3.0"
 rubato = "0.15.0"
 futures-util = "0.3.30"
 tokio = { version = "1", features = ["full"] }
-wgpu = { version = "0.19.1", features = ["naga-ir"] }
 schemars = { git = "https://github.com/membraneframework-labs/schemars", rev = "a5ad1f9", features = [
     "preserve_order",
 ] }
+shared_memory = "0.12.4"
+wgpu = { version = "0.19.1", default-features = false, features = [
```

Comment:
What are you actually changing here, is this disabling some default features, if so which?

Reply:
By default `wgpu` has `webgpu` feature enabled which has higher priority than `webgl` feature. Currently, compositor does not run on `webgpu` due to lack of support for push  constants on `webgpu` (they are not emulated like on `webgl`)

Reply:
Also, I added `fragile-send-sync-non-atomic-wasm` feature, which makes the wgpu types `Send` (by default they are not `Send` and `Sync` on wasm). This feature is slightly unsafe because on wasm those types should not  be shared between threads and it removes the compile time checks.

But, since we target web browsers which run the app on single thread it should not be problem for us. `LayoutProvider` needs to be  `Send` because `compositor_pipeline` uses threads. We could also solve it by making 2 `LayoutProvider`s (one Send and the other one non-Send) and enabling one based on `#[cfg(target_arch = "wasm32")]`. I decided to use the feature because it was just the easier way